ocs_ci/helpers/managed_services.py

```diff
@@ -19,7 +19,7 @@
 log = logging.getLogger(__name__)
 
 
-def verify_provider_topology():
+def verify_provider_topology(size=None):
     """
     Verify topology in a Managed Services provider cluster
```

Comment:
Please update the docstring with information on possible values of 'size'

Reply:
If I understand correctly, the only supported values of size variable are "4", "8", "12", "16" and "20". If that is true, I believe this information should be reflected in the docstring.